README.md

```diff
@@ -39,3 +39,9 @@ register_validation \
     "my_custom_validaton" \
     "<description>:"
 ```
+
+## Usage
+Navigate to the root folder you would like to evaluate and execute the following command
+```sh
+is-my-code-great
```

Comment:
maybe also worth to metion that you can specify a path like `is-my-code-great ~/dev/my_repo/`?